@here/harp-mapview/lib/MapView.ts

```diff
@@ -2554,8 +2554,12 @@ export class MapView extends THREE.EventDispatcher {
                 this.m_tileDataSources.length &&
             !this.m_updatePending &&
             !this.animating &&
+            !this.cameraIsMoving &&
+            !this.m_animatedExtrusionHandler.isAnimating &&
             this.m_textElementsRenderer !== undefined &&
```

Comment:
If ` this.m_textElementsRenderer === undefined` the event will never be triggered. Is this expected behavior (same for poiTableManager)?

Reply:
m_poiTableManager is created properly in the constructor, so be check for undefined is superfluous (fixing that).
m_textElementsRenderer may be undefined, but _will_always_ be created during rendering. The test for "defined" is to make the compiler happy.